src/main/java/com/adobe/cq/testing/client/security/AbstractAuthorizable.java

```
@@ -110,8 +109,18 @@ public String getUserPropertiesJsonAsString(int... expectedStatus) throws Client
         return client.doGet(getHomePath() + SELECTOR_USERPROPERTIES + ".json", expectedStatus).getContent();
     }
 
-    public boolean exists() throws ClientException {
-        return client.exists(getHomePath());
+    public boolean exists() {
```

any reason why this `exists()` swallows the exception and the other throws it?